src/app/archive/archive_lib/processor.ml

```diff
@@ -478,7 +488,7 @@ module User_command = struct
             , created_token )
           , user_command_id )
       in
-      user_command_id
+      (user_command_id, (fee_payer_balance, source_balance, receiver_balance))
```

Comment:
I missed this before (and indeed wrote the original incorrect code), but the sql update above should point at the `blocks_user_commands` table, with the fields moved to there.

For context: we may store multiple candidate blocks with the same transaction in them, and so these status fields are a property of the transaction being applied *in a particular block* rather than a property of the transaction itself. For example, if a transaction is applied in one block and creates the receiver account, but in another block it doesn't (because another previous transaction in that block/chain already created it) then the statuses will be different. As written, we only store one of these statuses, arbitrarily choosing between them based on which was last to be included.

Reply:
good catch